Makefile.am

```diff
@@ -1,10 +1,11 @@
+ACLOCAL_AMFLAGS = -I m4
```

Comment:
I think that what you are looking for is `ac-helpers`. Not sure how up to date it is, and I think it is fine to rename it to `m4`.